Sparked.TestDataClient/Sparked.TestDataClient.csproj

```diff
@@ -1,15 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>net70</TargetFramework>
-    <ImplicitUsings>enable</ImplicitUsings>
-    <Nullable>enable</Nullable>
-    <AssemblyName>TestDataClient</AssemblyName>
-  </PropertyGroup>
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>net80</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+        <AssemblyName>TestDataClient</AssemblyName>
+    </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Hl7.Fhir.R4" Version="5.6.1" />
-  </ItemGroup>
+    <ItemGroup>
+        <PackageReference Include="Hl7.Fhir.R4B" Version="5.11.1" />
```

Comment:
@heathfrankel could you please advice here?

May update to r4b cause some possible issues?
As far as I know au-core is based on r4.